README.md

```diff
@@ -71,7 +71,9 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I think the answer here is a helper method. Model methods would be related to the database, and we don't want to change any data - just how it's displayed in the browser. I think you could probably achieve the text color manipulation via a controller method, but based on some of our classes I think we want to keep the controller 'skinny'. By the process of elimination, I think helper methods are the way to go - so there would be some methods in the erb file to handle the color based on rating.
```

Comment:
:white_check_mark:  Correct! You don't define helper methods in the views, but you *use* them in the views.